### DISCUSSION TITLE
✅ 🔗 Loop grid with Woocommerce categories (Duplicate of #21372)

### DISCUSSION THREAD

**Discussion by QuentinVillanova** [elementor/elementor#22672] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I often need for my websites to create custom dynamics categories display.
The perfect widget for that is the Loop Grid.
But you can only use "posts" and "products" in Loop Grid.

### Describe the solution you'd like

The best solution for me is to include Woocommerce "categories" in the Loop Grid display type.

### Describe alternatives you've considered

The only alternative I've found is to set categories one by one on my page, but it's absolutely not dynamic !

**Comment by nicholaszein** [comment]
Hi @QuentinVillanova!

We are happy to announce that this feature will be released with Elementor Pro version 3.21, which is currently in beta testing, and expected to be released next week, April 15th!

We would love to hear your feedback about it! Please try it out and let us know! 💪 

Kind regards

**Comment by nicholaszein** [comment]
Hey @QuentinVillanova!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.21.0! 🥳 

You can now use the Loop Grid and Loop Carousel to display Taxonomies, together with their custom meta such as ACF images, using your own designs!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
